packages/auth-middleware/src/const.js

```diff
@@ -0,0 +1,2 @@
+export const SET_TOKEN = '@@authMiddleware_SET_AUTH_TOKEN';
```

Comment:
Maybe name the action types in the [ducks convention](https://github.com/erikras/ducks-modular-redux): `react-packages/auth/SET_TOKEN`

Also we can consider using the whole pattern in this particular case @marcin-piela @aherok ? If not we the `action-types.js` is a better filename.